pkg/support/dump_others.go

```diff
@@ -60,18 +62,51 @@ func (d *agentDumper) dumpIPTables(basedir string) error {
 }
 
 func (d *agentDumper) dumpIPToolInfo(basedir string) error {
-	dump := func(name string) error {
-		output, err := d.executor.Command("ip", name).CombinedOutput()
+	dump := func(name string, args ...string) error {
+		cmdArgs := append([]string{name}, args...)
+		output, err := d.executor.Command("ip", cmdArgs...).CombinedOutput()
 		if err != nil {
 			return fmt.Errorf("error when dumping %s: %w", name, err)
 		}
 		return writeFile(d.fs, filepath.Join(basedir, name), name, output)
 	}
-	for _, item := range []string{"route", "link", "address"} {
+	for _, item := range []string{"link", "address"} {
 		if err := dump(item); err != nil {
 			return err
 		}
 	}
+	if err := dump("rule", "show"); err != nil {
+		return err
+	}
+	if err := dump("route", "show", "table", "all"); err != nil {
+		return err
+	}
+	
+	if err := d.dumpInterfaceConfigs(basedir); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (d *agentDumper) dumpInterfaceConfigs(basedir string) error {
+	interfaces, err := net.Interfaces()
```

Comment:
I don't think we want to consider *all* interfaces. There can be a lot of them because there is one per local Pod.
We should include the following interfaces:
- the host gateway (`antrea-gw0` by default but is configurable, so it should not be hardcoded, instead you need to use `HostGateway` from the config)
- `antrea-egress0`
- `antrea-ingress0`
- `antrea-ext.<VLAN>` (there can be several such interfaces, only the prefix stays the same)

I also think this should be best effort, if we fail to get the information (maybe because of a permission issue), let's log the error + write the error to the support bundle, but let's not fail the bundle operation.